src/App.js

```diff
@@ -1,23 +1,17 @@
-import styled from 'styled-components/macro';
+import styled from 'styled-components/macro'
 import TutorialTemplate from './TutorialTemplate'
 
-
-
 export default function App() {
   return (
-      <PageLayout>
-        <TutorialTemplate/>  
-        <TutorialTemplate/>  
-        <TutorialTemplate/> 
-      </PageLayout>
-  );
+    <PageLayout>
+      <TutorialTemplate />
+    </PageLayout>
+  )
 }
 
 const PageLayout = styled.div`
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
   position: relative;
   margin: 0;
```

Comment:
You've got a grid going on, might as well use it to position your content ;)

Reply:
I now! WIP ;)